src/vivarium_gates_nutrition_optimization_child/constants/paths.py

```diff
@@ -14,14 +14,14 @@
 IFA_GA_SHIFT_DATA_DIR = RAW_DATA_DIR / "ifa_gestational_age_shifts"
 MMS_GA_SHIFT_1_DATA_DIR = RAW_DATA_DIR / "mms_gestational_age_shifts/shift1"
 MMS_GA_SHIFT_2_DATA_DIR = RAW_DATA_DIR / "mms_gestational_age_shifts/shift2"
-SQLNS_RISK_RATIOS = RAW_DATA_DIR / "sqlns_risk_ratios.csv"
-WASTING_TRANSITIONS_DATA_DIR = RAW_DATA_DIR / "wasting_transition_rates"
+SQLNS_RISK_RATIOS = RAW_DATA_DIR / "2021_sqlns_effects_by_location.csv"
```

Comment:
would it be easier if I saved these as separate csvs for each location rather than a single csv with a location column?

Reply:
Hey Ali - maybe yes, that matches the format of the other data inputs better and so we can follow the same code. However, either works quite easily so its not particularly important. When we create the Nigeria and Pakistan versions, lets switch to 3 files.

Reply:
Sounds good!